modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -114,4 +117,12 @@ export class Lightning implements ILightning {
       throw new Error(`error(s) parsing response body: ${errors}`);
     });
   }
+
+  public async decodeLnurlPay(lnurl: string): Promise<DecodedLnurlPayRequest> {
+    return await decodeLnurlPay(lnurl);
+  }
+
+  public async fetchLnurlPayInvoice(params: LnurlPayParams): Promise<string> {
+    return await fetchLnurlPayInvoice(params, this.wallet.coin());
```

Comment:
i believe we need those keywords as `decodeLnurlPay` and `fetchLnurlPayInvoice` utils return `Promise`

Reply:
Yes but they can return promises without being `async`, and the promises don't need to be `await`ed here. Functionally it doesn't make much difference, I just generally try not to declare functions as `async` unless they have to be.

Reply:
I agree, we don't need to `await` it at the end. Since we are returning a promise, after `await` we need to wrap it again in a `Promise` compared to just returning the promise as is, but its really a nit